src/index.ts

```diff
@@ -13,15 +20,27 @@ const hasFeatureSupport = hasProperties(window, requiredFeatures);
 const state: Fastly = {
   client: {
     hasFeatureSupport
-  }
+  },
+  // The are certain scenarios (such as the demo website) in which the gloabl
+  // FASTLY and the results array already exists.
+  results: (window.FASTLY && window.FASTLY.results) || []
```

Comment:
If `window.FASTLY` is guaranteed to be there, you might not need the first truthy condition?
Otherwise, update the type to be `FASTLY?: Fastly` ?

Reply:
:+1: It's not guaranteed to be there, so have updated the type to reflect this.